tools/qlog/src/lib.rs

```diff
@@ -101,16 +86,13 @@
 //!
 //! ## Adding events
 //!
-//! Qlog Events are added to [`qlog::Trace.events`].
+//! Qlog [`Event`] objects are added to [`qlog::Trace.events`].
 //!
-//! It is recommended to use the provided utility methods to append semantically
-//! valid events to a trace. However, there is nothing preventing you from
-//! creating the events manually.
-//!
-//! The following example demonstrates how to log a QUIC packet
-//! containing a single Crypto frame. It uses the [`QuicFrame::crypto()`],
-//! [`packet_sent_min()`] and [`push_event()`] methods to create and log a
-//! PacketSent event and its EventData.
+//! The following example demonstrates how to log a qlog QUIC `packet_sent`
+//! event containing a single Crypto frame. It constructs the necessary elements
+//! of the [`Event`], the then appends it to the trace with [`push_event()`].
```

Comment:
I think this comment needs some reworking? Besides "the then", it later mentions `QuicFrame::crypto()` which was removed.

Reply:
good spot. I fixed "the then" and just removed the part about frame construction because it should be obvious how to instantiate a struct.